### PR TITLE
Fixes #12110 - updating puppet-pulp to use is_parent => true param

### PR DESCRIPTION
**Pull request by johnpmitsch**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -64,4 +64,6 @@
 
   $qpid_url = "amqp:ssl:${::fqdn}:5671"
   $candlepin_event_queue = 'katello_event_queue'
+
+  $is_parent = true
```
Comment (stbenjam) [inline manifests/params.pp]: Is this intended to be used somewhere?

Comment (johnpmitsch) [inline manifests/params.pp]: trying to update for this change https://github.com/Katello/puppet-pulp/pull/108

Comment (stbenjam) [inline manifests/params.pp]: params.pp is only if the parameter is actually a class parameter (in this case on katello  itself).

File: manifests/init.pp [hunk 2]
```diff
@@ -109,6 +109,7 @@
     enable_puppet         => true,
     enable_docker         => true,
     num_workers           => $num_pulp_workers,
+    is_parent             => true,
```
Comment (johnpmitsch) [inline manifests/init.pp]: @stbenjam so basically I would just need this?

**Comment by johnpmitsch** [comment]
Waiting on https://github.com/Katello/puppet-pulp/pull/108 to be merged

File: manifests/init.pp (hunk 2 above)
Comment (stbenjam) [inline manifests/init.pp]: Yup, that's it. Only need the params.pp in the case you have in the other PR, when adding it to the class's parameters list.

**Comment by stbenjam** [comment]
Looks good to me,  APT

**Comment by ehelms** [comment]
Thanks @johnpmitsch ! Please do the same to puppet-katello_devel

**Pull request merged**